### ISSUE TITLE
Label for and id are invalid HTML in Selects tab

### ISSUE THREAD

**Issue by mathisscott** [vmware/clarity#1921]
URL: http://clarity-ks.surge.sh/selects

Tools: Web Accessibility Toolbar (WAT2.0) / ARIA Markup

Reproduced Steps:
1. Login Clarity Kitchen Sink with web browser.
2. Go to Selects tab.
3. Check label and id in Selects tab.

Actual Results:
The label for associating "Text field paired with a select” text only include id="selects_2".
The label for and id associating “Multi Select” text and “Disabled select” text are invalid HTML.
For more details, please refer to "Selects-NoMarkup.zip".

Expected Results:
The label for associating "Text field paired with a select” text should include id="selects_3".
The label for and id associating “Multi Select” text and “Disabled select” text must be identical (case sensitive).

Fix Suggestion:
Set label for property to “label for="selects_2, selects_3"” for "Text field paired with a select” text.
Set label for property to “label for=" selects_5"” for "Multi Select” text.
Set label for property to “label for=" selects_6"” for "Disabled select” text.

Violation of Section 508 Standards: 
Section 1194.22 (n) When electronic forms are designed to be completed on-line, the form shall allow people using assistive technology to access the information, field elements, and functionality required for completion and submission of the form, including all directions and cues.

User Impact: Blind.

Screen shot:
1.	Selects-NoMarkup.png

![image.png](https://images.zenhubusercontent.com/582f4cffb9dc3c6220222f57/595803d3-dbdc-4964-a6f4-4ee83e4fe9a2)

NOTE: this may also be an issue on the website

**Comment by Jinnie** [comment]
All is clear for Multi select and Disabled select. 
But the suggested `label for="selects_2, selects_3"` is invalid according to W3C standards:
https://www.w3.org/wiki/Html/Elements/label
Please, provide reference for the suggested syntax.
According to my research, either we need a fieldset with a legend for the "1 label, 2 inputs" scenario, or we need an offscreen hidden label.
